packages/coinstac-api-server/src/data/schema.js

```diff
@@ -164,7 +164,7 @@ const typeDefs = `
     leaveConsortium(consortiumId: ID!): Consortium
     removeComputation(computationId: ID): Computation
     removeUserRole(userId: ID!, table: String!, doc: String!, role: String!): User
-    saveActivePipeline(consortiumId: ID, activePipelineId: ID): String
+    saveActivePipeline(consortiumId: ID, activePipelineId: ID): JSON
```

Comment:
whats up with this change?

Reply:
check **saveActivePipeline** in resolvers.
It returns updated result and it is JSON not string.